packages/wasm-sdk/index.html

```
@@ -12,11 +12,83 @@
+
+    .status-banner.error {
+      background-color: #f44336;
+      color: white;
+    }
+
+    .status-banner.loading {
+      background-color: #ff9800;
+      color: white;
+    }
   </style>
 </head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)